src/services/challengeApi.tsx

```diff
@@ -20,19 +21,24 @@ export const getOngoingChallenges = async (
   filterUser: boolean,
   sortDesc: number,
 ) => {
-  const response = await api.get(
+  const { data } = await api.get<{ data: Array<ChallengeResponseType> }>(
     `${route}/?showOngoing=true&sortDesc=${sortDesc}&filterUser=${filterUser}`,
   );
-  return response.data;
+  return data.data as Array<ChallengeResponseType>;
 };
 
 export const getPastChallenges = async (
-  filterStatus: ChallengeStatusEnum | undefined,
+  filterStatus: `${ChallengeStatusEnum}` | undefined,
   sortDesc: number,
 ) => {
   const status = filterStatus ? `&filterStatus=${filterStatus}` : ``;
```

Comment:
This should be:

`  const status = filterStatus ? filterStatus : ``;`

Reply:
why?